src/promptflow-tools/promptflow/tools/open_source_llm.py

```diff
@@ -821,9 +990,16 @@ def call(
         content_formatter = ContentFormatterFactory.get_content_formatter(
             model_family=self.model_family,
             api=api,
-            chat_history=prompt
+            chat_history=prompt,
+            endpoint_url=self.endpoint_uri
         )
 
+        if isinstance(content_formatter, ServerlessLlamaContentFormatter):
```

Comment:
@georgedyre , can you add test cases for serverless llama. To keep it simple create a custom connection with serverless endpoint details and use that in your test.